### ISSUE TITLE
Issue #960 details: Python Interpreter Window, font color problem

### ISSUE THREAD

**Issue by ocarillo** [pyscripter/pyscripter#961]
I tested with several versions including the last one 3.6.1 and I always have the same problem. I open the program, change the Output item as you suggest, and the color of the output text remains unchanged. I took some captures to show the problem:

In picture one I change the color of the output and set to clLime (#00FF00)
In picture two, you can see that the color of the font remains in default color #008080 when type something
In picture three, I open the dialog again and the color doesn't remain in clLime, It has changed to the default color.

![image](https://user-images.githubusercontent.com/50459080/61687757-76329d80-ad23-11e9-8ca2-693d1a73fa14.png)

![image](https://user-images.githubusercontent.com/50459080/61687789-88144080-ad23-11e9-9c98-4d4b548dfbee.png)

![image](https://user-images.githubusercontent.com/50459080/61687815-9cf0d400-ad23-11e9-97b6-04949bd99265.png)

_Originally posted by @ocarillo in https://github.com/pyscripter/pyscripter/issues/960#issuecomment-514073468_

**Comment by pyscripter** [comment]
In the steps descriped in  #960 Are you doing step 5

> 
5.  Click some other element for the color to be entered.

before pressing OK?

**Comment by ocarillo** [comment]
Yes, I followed your instructions and the problem remains. I changed the color of the Output item and the color of Traceback item. Then I pressed OK. If I open the Dialog again, the Traceback item has the color changed but the Output item doesn't, remaining with de default color.

![image](https://user-images.githubusercontent.com/50459080/61704710-9de62d80-ad44-11e9-8a39-7fb2c6f68cad.png)

 It seems to be only with the Output item. The rest of items seems to be working OK (I tested some of then and no problems detected)

**Comment by pyscripter** [comment]
Strange.  It works here fine.

I see in your picture that "Hi John" is shown with Lime color.   Why you are saying that Output is not changed?

**Comment by ocarillo** [comment]
The pictured attached is the first try. I attaching you the two steps. First picture, I change the Output item to #00FF00 and the Traceback item to  #9900FF. Then I press OK. Then I open the dialog again and you can see in the second picture that the Traceback item has the right color, but the Output item has set back to #008080. I running Pyscripter 3.6.1 portable, but I have had the same problem in different versions, for example in 3.6 (portable and not portable and previous). I have this behaviour in different computers.

![image](https://user-images.githubusercontent.com/50459080/61705362-ece09280-ad45-11e9-938d-5195993c06fb.png)

![image](https://user-images.githubusercontent.com/50459080/61705419-04b81680-ad46-11e9-80ca-44b6ccebefb2.png)

**Comment by pyscripter** [comment]
I was able to reproduce the problem with PyScripter 3.6.1 following the exact instructions.  (change Output and Traceback colors together).  
However with the active version control branch I am not able to reproduce the problem.  So it appears that some change introduced fixed the issue, 
Could you please try the executable in this [zip file](https://drive.google.com/file/d/1ggLY1q_Bk-yI5cKfaXQKkwMnmyv52Y3m/view?usp=sharing) and report whether you are still getting the same problem?  It is for PyScripter 32 bits.

**Comment by ocarillo** [comment]
Hi,

The problem persists with your executable attached. This is the steps to reproduce de issue (the same behaviour in all machines and all versions I tested):

1. On Python Interpreter window -> Right mouse click -> Interpreter Editor options ...-> Syntax colors tab -> Select "Output" item -> Change to any color (you will see the "Hi John" text with the selected color) -> Press "OK" button.

2. Open again the window: On Python Interpreter window -> Right mouse click -> Interpreter Editor options ...-> Syntax colors tab.  Then, you will see that the "Hi John" text is again with the original color "#008080", not with the selected one.

 This behaviour only happens with the "Output" item. Any other item, it keeps the color configuration without problems.

**Comment by pyscripter** [comment]
Could you please check again with PyScripter 3.6.2?  Does the error persist if you delete/rename pyscripter.ini?

**Comment by ocarillo** [comment]
I still have the same problem with 3.6.2. See the animated gif to see the problem. I tested the portable version 3.6.2, but the problem persists in all versions I tested.

![Screen-Recording-_30-12-2019-9-30](https://user-images.githubusercontent.com/50459080/71574400-d73b7a00-2ae8-11ea-8809-c6a212315bff.gif)

**Comment by Chrysalida** [comment]
> I still have the same problem with 3.6.2. 

Dear author, have you solved the problem? I have exactly the same issue with my interpreter output color (Pyscrypter 3.6.3), and it would be great to know if there is a solution somewhere.

**Comment by Chrysalida** [comment]
the worst thing is that the output color remains the same no matter what color scheme I choose, no matter what color the output should be accordingly to this scheme. Moreover, to change the output background color isn't possible too! 
The output just remains that dark cyan on the black, always!

**Comment by pyscripter** [comment]
Can you please try the following:
-  Use the latest PyScripter 3.6.3.  (just to make sure we use the same version).
-  Find pyscripter.ini in your %APPDATA%\pyscripter directory (%APPDATA % is the value of the environment variable) and rename it.  Then restart Pyscripter.

In other words, open a command prompt and do:
cd %APPDATA%\PyScripter
ren PyScripter.ini PyScripter.ini.bak

-  Then start PyScripter again, customize the Editor colors theme (Editor Options, Color Theme, Apply Theme)
-  Finally try again to customize the output foreground and background colors.

**Comment by Chrysalida** [comment]
> Can you please try the following:
> ...

Still no effect after following yor instructions step-by-step exactly, but thank you anyway. 
Actually, the unchangeable color wouldn't be a problem at all if it only was a little brighter - this dark cyan is just hard readable.

![PyIssue](https://user-images.githubusercontent.com/54242060/92894753-8758d500-f423-11ea-8dee-bd50c9d73339.png)

**Comment by pyscripter** [comment]
@Chrysalida Could you please post here your pyscripter.ini file in the %APPDATA%/Pyscripter folder?

**Comment by ocarillo** [comment]
Hi,

I solved the issue adding the following lines manually to pyscripter.ini file:

[Highlighters\Intepreter\OutputAttri]
Foreground=clLime

Doing this, the color font changes, (from menu, not working). It appears to be a problem only with this attribute (OutputAttri), as the rest of colors seems to work ok.

**Comment by pyscripter** [comment]
@ocarillo Thanks I will have another look to see why.

**Comment by Chrysalida** [comment]
> @Chrysalida
> Did you rename your Pyscripter.ini file in the %APPDATA%/Pyscripter folder?
Sure, I did even twice)

[PyScripter_ini.zip](https://github.com/pyscripter/pyscripter/files/5238169/PyScripter_ini.zip)

Here is my .ini - the fresh one.

**Comment by pyscripter** [comment]
@Chrysalida I did try to reproduce this issue but I can't.   Whenever I change the Output color in the Interpreter editor options, the change is reflected in the Interpreter Windows and saved and restored properly.   I am not sure why you are facing this issue on your side.

You can implement the workaround by @ocarillo.  i.e. edit the PyScripter.ini file and add (or change if it exists) [Highlighters\Intepreter\OutputAttri].

```
[Highlighters\Intepreter\OutputAttri]
Foreground=clLime
```

In the forthcoming version I will be using #00BBFF (brighter than the current #008888 Teal) as default which corresponds to 
Foreground=16759552

**Comment by pyscripter** [comment]
Is this still an issue with version 4?

**Comment by ocarillo** [comment]
Hi,

Let me answer the question. I confirm that the problem persists on new version 4.0.0.6 running on win 10 64 bit. It seems you haver forgotten to create the pyscripter.ini file with the "Highlighters\Intepreter\OutputAttri" tag inside. The rest or tags seems to work fine, just this one.

**Comment by pyscripter** [comment]
I am closing this since:
-  There is a straightforward workaround
- I could not reproduce it.
- The default output color has been changed to a more suitable (bright) one
- OutputAttri is now included in the default Pyscripter.ini file (which was the workaround suggested).
